js/menus/menuTemplate.js

```diff
@@ -65,7 +65,23 @@ const template = [{
         }
     },
     {
-        label: 'Open Crashes Directory',        
+        label: 'Set Downloads Directory',
+        click() {
+            electron.dialog.showOpenDialog({
+                title: 'Select Downloads Directory',
+                buttonLabel: 'Select',
+                properties: ['openDirectory', 'createDirectory']
+            }, (filePaths) => {
+                if (!filePaths || !Array.isArray(filePaths) || filePaths.length < 1) {
```

Comment:
So, does this happen when the user cancels the selection?

Reply:
Yes, and when nothing is selected. It's just an extra check to ensure the sanctity of the app.